django/core/validators.py

```diff
@@ -201,12 +202,10 @@ def __call__(self, value):
         if (domain_part not in self.domain_whitelist and
                 not self.validate_domain_part(domain_part)):
             # Try for possible IDN domain-part
-            try:
+            with suppress(UnicodeError):
                 domain_part = domain_part.encode('idna').decode('ascii')
                 if self.validate_domain_part(domain_part):
```

Comment:
Not sure this should be in the context manager. Technically it would probably be in the else of try/except/else.